### PR TITLE
feat: put update and recheck interval values to .env file if present

### PR DESCRIPTION
**Pull request by mogita**
I was about to submit an issue but as I wrote I seemed to have an idea of how the code should be changed. Please read on for the reason of this PR. The script and README.md were updated correspondingly. Thank you in advance for reviewing!

Original issue draft:

**Is your feature request related to a problem? Please describe.**
Running the version 3.7 of this image with `NOMINATIM_REPLICATION_UPDATE_INTERVAL` and `NOMINATIM_REPLICATION_RECHECK_INTERVAL` set won't affect the actual behavior of the update and check interval as they look to be hardcoded in the `.env` file.

**Describe the solution you'd like**
Update the corresponding variables in the `.env` file with a user specified value or use the default when not provided.

**Describe alternatives you've considered**
I think setting a readonly configMap and point to the `.env` file might override this hardcode. But it looks ugly.

**Additional context**
https://github.com/mediagis/nominatim-docker/blob/master/3.7/conf.d/env#L2

### CODE REVIEW

File: 3.7/README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ The following environment variables are available for configuration:
 
   - `PBF_URL`: Which OSM extract to download. Check https://download.geofabrik.de
   - `REPLICATION_URL`: Where to get updates from. Also available from Geofabrik.
+  - `REPLICATION_UPDATE_INTERVAL`: How often upstream publishes diffs (in second, default: `86400`)
```
Comment (leonardehrenfried) [inline 3.7/README.md:28]:
```suggestion
  - `REPLICATION_UPDATE_INTERVAL`: How often upstream publishes diffs (in seconds, default: `86400`)
```

File: 3.7/README.md [hunk 2]
```diff
@@ -25,6 +25,8 @@ The following environment variables are available for configuration:
...
+  - `REPLICATION_RECHECK_INTERVAL`: How long to sleep if no update found yet (in second, default: `900`)
```
Comment (leonardehrenfried) [inline 3.7/README.md:29]:
```suggestion
  - `REPLICATION_RECHECK_INTERVAL`: How long to sleep if no update found yet (in seconds, default: `900`)
```

**Review by leonardehrenfried — APPROVED** [review]
This is a good PR - I approve.

Thanks!

**Comment by philipkozeny** [comment]
Nice one, thanks!

I would like to change the way you set the values though to keep it consistent with the rest of our `config.sh` though. If you take a look at the Postgres tuning we have the default values set and just run `sed` if the environment variable is set (https://github.com/mediagis/nominatim-docker/blob/d657ec38855541e9a5a9fbe3eae1043f844035ac/3.7/config.sh#L34).  Can you do that? Will merge after that, rest looks fine, thanks a lot!

**Comment by mogita** [comment]
> Nice one, thanks!
> 
> I would like to change the way you set the values though to keep it consistent with the rest of our `config.sh` though. If you take a look at the Postgres tuning we have the default values set and just run `sed` if the environment variable is set (
> 
> https://github.com/mediagis/nominatim-docker/blob/d657ec38855541e9a5a9fbe3eae1043f844035ac/3.7/config.sh#L34
> 
> ). Can you do that? Will merge after that, rest looks fine, thanks a lot!

Code changed accordingly. Please have a look. Thanks!

**Comment by philipkozeny** [comment]
@all-contributors please add @mogita for docs and code.

**Pull request merged**